LDKNodeMonday/Utilities/Constants.swift

```diff
@@ -11,6 +11,9 @@ import SwiftUI
 
 struct Constants {
 
+    static let satsPerBtc: UInt64 = 1_00_000_000
```

Comment:
If these can be added as an extension on UInt64 in `UInt64+Extensions` potentially

Reply:
Done